gmso/formats/mol2.py

```diff
@@ -51,165 +51,88 @@ def from_mol2(filename, site_type="atom"):
     # save the name from the filename
     with open(mol2path, "r") as f:
         line = f.readline()
-        while f:
-            # check for header character in line
-            if line.strip().startswith("@<TRIPOS>"):
-                # if header character in line, send to a function that will direct it properly
-                line = _parse_record_type_indicator(
-                    f, line, topology, site_type
+        with open(filename, "r") as f:
+            fcontents = f.readlines()
+        sections = dict()
+        key = "meta"
```

Comment:
I think this line needs to be 
section_key = "meta"

Reply:
good catch!